viite-UI/src/view/LinkPropertyLayer.js

```diff
@@ -518,6 +526,14 @@
             floatingMarkerLayer.getSource().addFeature(marker);
         });
 
+        var directionRMIds = _.pluck(directionRoadMarker, 'linkId');
```

Comment:
Maybe we need stop using the pluck and use the map instead, because, pluck was removed from lodash 4.